packages/legend-query/src/stores/QueryBuilderExplorerState.ts

```diff
@@ -122,6 +133,32 @@ export class QueryBuilderExplorerTreePropertyNodeData extends QueryBuilderExplor
   }
 }
 
+export class QueryBuilderExplorerTreeSubTypeNodeData extends QueryBuilderExplorerTreeNodeData {
+  subClass: Class;
```

Comment:
do me a favor and scan in your code, `subClass` -> `subclass` - follow the spirit of https://github.com/finos/legend-studio/pull/775